docs/nodes/communityNodes.md

```diff
@@ -0,0 +1,28 @@
+# Community Nodes
+
+These are nodes that have been developed by the community for the community. If you're not sure what a node is, you can learn more about nodes [here](overview.md).
+
+If you'd like to submit a node for the community, please refer to the [node creation overview](overview.md).
+
+To download a node, simply download the `.py` node file from the link and add it to the `/app/invocations/` folder in your Invoke AI install location. Along with the node, an example node graph should be provided to help you get started with the node. 
```

Comment:
This is misleading. The  `app/invocations/` folder is part of the source code tree, actually located under `invokeai/app/invocations` . Most users will never see this folder because they are performing a wheel or pip install.

There is no `app/invocations/` folder in the InvokeAI root directory, and as far as I can tell there's no code that looks for .py files located there. However, this would be a nice extension mechanism.

Just change the wording to indicate that this directory is in the source tree.

Reply:
Added `invokeai` to the path